pkg/manager/manager.go

```diff
@@ -139,6 +139,23 @@ type Options struct {
 	// value only if you know what you are doing. Defaults to 10 hours if unset.
 	// there will a 10 percent jitter between the SyncPeriod of all controllers
 	// so that all controllers will not send list requests simultaneously.
+	//
```

Comment:
Nit: Could you mention that this setting applies to all controllers (which is why I think using it is bad but we don't have a nice `Source` that would make this easy on controller-level) and that it works by creating an artificial ADD event?

Reply:
```suggestion
	//
	// This applies to all controllers.
	//
```